src/index.js

```diff
@@ -239,8 +239,8 @@ function needsMoneyComponent(location) {
 }
 
 function noIdNeededComponent(location) {
-  if(location.noIdNeeded === "TRUE") {
-    if (location.someInfoRequired === "TRUE") {
+  if(location.noIdNeeded === true) {
```

Comment:
These are actual booleans from backend now, instead of google sheets translating them to strings